iroha_p2p/src/lib.rs

```diff
@@ -6,14 +6,20 @@ use std::{io, net::AddrParseError};
 
 use iroha_derive::FromVariant;
 use iroha_error::{derive::Error, error};
-pub use network::{Connect, Network, Post, Received};
+pub use network::{Connect, NetworkBase, Post, Received};
+use parity_scale_codec::{Decode, Encode};
+use ursa::{encryption::symm::prelude::ChaCha20Poly1305, kex::x25519::X25519Sha256};
 
-mod message;
-mod network;
-mod peer;
+/// Network is a main p2p start point.
+pub mod network;
+/// Peer is an endpoint to another node.
+pub mod peer;
```

Comment:
Guess peer does not needed to be public. It can be just `pub(crate)` or smth like that

Reply:
It's needed for tests. `PeerId`, for example.